Python/ceval.c

```diff
@@ -427,10 +427,16 @@ _PyEval_ReInitThreads(_PyRuntimeState *runtime)
     take_gil(tstate);
 
     struct _pending_calls *pending = &tstate->interp->ceval.pending;
+#ifdef HAVE_FORK
+    if (_PyThread_at_fork_reinit(&pending->lock) < 0) {
+        Py_FatalError("Can't initialize threads for pending calls");
+    }
+#else
     pending->lock = PyThread_allocate_lock();
```

Comment:
With _PyThread_at_fork_reinit() call, ``pending->lock = PyThread_allocate_lock()`` is no longer needed!